### ISSUE TITLE
Nagrand - Flying Over Waterfalls causes death by falling damage

### ISSUE THREAD

**Issue by Nogrial** [WoWManiaUK/Redemption#179]
Starting at 57,74 and flying north toward 55,67, while low to the water, you will eventually hit a random spot and die by falling damage. You get dismounted and are dead.
- Goblin
- Mount: Turbo-Charged Flying Machine

**Comment by Kalasbyxan** [comment]
Confirmed. Dead at 57.73 while flying on Turbo-Charged Flying Machine.

**Comment by wow-mania** [comment]
I could reproduce it only once after passing by loads of times.
This will be very time consuming to fix and since it's only a minor inconvenience I will mark it as "won't fix" for now.

**Comment by Keader** [comment]
Fixed, using #3753 method.
